rest/data/commit_queue.go

```diff
@@ -388,3 +488,30 @@ func GetAdditionalPatches(patchId string) ([]string, error) {
 		Message:    errors.Errorf("patch '%s' not found in commit queue", patchId).Error(),
 	}
 }
+
+func requireSigned(ctx context.Context, userRepo UserRepoInfo, prNum int) error {
```

Comment:
nit: can we call this something like `checkSignedCommit` or something?

Reply:
Done